tox.ini

```diff
@@ -29,7 +29,7 @@ commands = coverage erase
 install_command = pip install {opts} {packages}
 commands =
   pylint {[base]pkg_name} --ignore=tm_tokenize
-  black -v --diff --check {toxinidir}
+  black -v {toxinidir}
```

Comment:
Don't we want check here to find errors?

Reply:
I remove `--diff` and `--check` in my local env so that black can autoformat. By mistake committed those changes. Reveted in latest commit.